enterprise_access/apps/workflow/serialization.py

```diff
@@ -0,0 +1,38 @@
+"""
+Workflow Steps do work on objects, not on dicts.  The BaseInputOutput
+class below can be subclassed by attrs classes, so that every input/output
+class utilized by workflow steps can be counted on to have a from_dict()
+classmethod to generate an object from a dict, and a to_dict() method
+to turn an instance of an input/output class back into a dict (in other words
+to structure and unstructure, or to deserialize and then serialize).
+"""
```

Comment:
I need to write up an ADR about "why attrs/cattrs?" still, but here's some justification from the attrs docs: https://www.attrs.org/en/stable/why.html#dicts

Reply:
Thanks for the link to the docs. An ADR about "why attrs/cattrs?" would be helpful!